libpod/runtime.go

```diff
@@ -475,6 +475,8 @@ func NewRuntimeFromConfig(configPath string, options ...RuntimeOption) (runtime
 	runtime.config = new(RuntimeConfig)
 	runtime.configuredFrom = new(runtimeConfiguredFrom)
 
+	deepcopier.Copy(defaultRuntimeConfig).To(runtime.config)
```

Comment:
I wonder if we shouldn't just make NewRuntimeFromConfig share code with NewRuntime, only setting the config path... Keeping them separate seems to be pointless

Reply:
good idea.  Let's drop one of the two implementations.

Pushed a new version